frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardMergeBranchToParentAction.java

```diff
@@ -34,12 +34,12 @@ public IEnhancedLambdaLogger log() {
 
   @Override
   public @I18nFormat({}) String getActionNameForDisabledDescription() {
-    return getString("action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name");
+    return getString("action.GitMachete.BaseFastForwardMergeToParentAction.description-action-name");
```

Comment:
Yeah, CI failing, as expected

Reply:
Yeah, I forgot about this at this time :/